error-issue/src/types.ts

```diff
@@ -80,3 +80,58 @@ export interface ErrorReport {
   dailyOccurrences: number;
   stacktrace: string;
 }
+
+/**
+ * Types used in Stackdriver API requests.
+ * Note that other fields may be present, but only the ones relevant are
+ * included in the definitions below.
+ */
+export namespace Stackdriver {
```

Comment:
nit: reorder interfaces so that exported are grouped together

Reply:
Done.